### ISSUE TITLE
Bootstrap process does not modify authorized_keys

### ISSUE THREAD

**Issue by archony** [discourse/discourse_docker#17]
I had to manually do it:
echo ""mysshkey"" >> /root/.ssh/authorized_keys

./launcher ssh app worked only after I manually did this, authorized_keys wasn't even getting created. I was doing it as root.

**Comment by SamSaffron** [comment]
no longer needed, we use docker exec
